tests/integration/config.py

```diff
@@ -44,22 +44,30 @@
 #   - ''                        <-- do not configure proxy
 MK8S_NO_PROXY = os.getenv("MK8S_NO_PROXY")
 
-# MK8S_METALLB_SPEAKER_CHARM is the metallb-speaker charm for the MetalLB integration tests.
```

Comment:
These look like two PRs in one. Are the metallb fixes related to the release process? If not could we please move them to a separate PR?